backend/src/services/integration-auth/integration-sync-secret.ts

```diff
@@ -1938,9 +1938,9 @@ const syncSecretsCircleCI = async ({
     })
   ).data;
 
-  let orgSlug: string | null = null;
+  let projectSlug: string | null = null;
   if (!integration.owner) {
-    orgSlug = `${circleciOrganizationDetail[0].slug}/${integration.app}`;
+    projectSlug = `${circleciOrganizationDetail[0].slug}/${integration.app}`;
```

Comment:
It looks like you are still choosing the first org name from the list of orgs that get fetched via the collaborations API. I don't think this will work because the slug is in the fortmat of `vcs-slug/org-name/repo-name`. We need to to know which org to scope it to. Here is a example of what the collaborations API retruns this. It already has has the vcs-slug, the org id + name all it is missing is the project name which we have. 

I think it will be better if we get the org of the project the user selected so that we can create this project slug. What do you think?


<img width="647" alt="CleanShot 2024-09-05 at 08 41 36@2x" src="https://github.com/user-attachments/assets/2db8428e-f907-440c-9e63-8a5d0bd2f49d">